src/redux/actions/getDeployments.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+import { GET_DEPLOYMENTS } from './actionTypes';
+import { API_BASE_URL } from '../../config';
+
+const getDeployments = (clusterID) => (dispatch) => {
+  axios.get(`${API_BASE_URL}/clusters/${clusterID}/deployments`)
+    .then((response) => {
+      console.log(response.data.data.deployments);
```

Comment:
Kindly remove the console logs, we cant have these and the debugger lines in our production code.